instrumented/App.js

```diff
@@ -0,0 +1,17 @@
+import './App.css';
+import { Component } from 'react';
+import Header from './Components/Header/Header';
+import MainContainer from './Components/MainContainer/MainContainer';
+
+class App extends Component {
+  render = () => {
+    return (
+      <div className="App">
+        <Header />
+        <MainContainer />
+      </div>
+    );
+  }
+}
+
+export default App;
```

Comment:
I didn't see this file being used anywhere in the tests. Can you please explain what this file is for and if it's needed?

Reply:
'Explained' [here](https://github.com/mi-voz-mi-voto/mi-voz-mi-voto/pull/30#discussion_r804866273)